inc/class-statify-frontend.php

```diff
@@ -247,7 +247,8 @@ private static function is_bot( $user_agent ) {
 	 */
 	private static function _is_internal() {
 		// Skip for preview, 404 calls, feed and search access.
```

Comment:
This line could be expanded with the Favicon check info

Reply:
Right. If we document self explaining code, we should do it completely :wink:

Reply:
Or remove completely as this is a little bit silly ;)